File: deal/src/main/java/ru/artemev/deal/DealApplication.java
package ru.artemev.deal;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;

@SpringBootApplication
@EnableFeignClients("ru.artemev.deal")
public class DealApplication {

  public static void main(String[] args) {
    SpringApplication.run(DealApplication.class, args);
  }
}

File: deal/src/main/kotlin/ru/kit/deal/DealApplication.kt
package ru.kit.deal

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.openfeign.EnableFeignClients

@SpringBootApplication
@EnableFeignClients("ru.kit.deal")
class DealApplication

fun main(args: Array<String>) {
    runApplication<DealApplication>(*args)
}
